### PR TITLE
Audio Switch Component - Adjusted the Properties section

### PR DESCRIPTION
**Pull request by LB-ArturZieba**
Signed-off-by: Artur Zięba <86952082+LB-ArturZieba@users.noreply.github.com>

## Change summary

* Audio Switch Component (https://www.o3de.org/docs/user-guide/components/reference/audio/switch/):
   * Adjusted the "Audio Switch Properties" section to contain a table, similar to https://www.o3de.org/docs/user-guide/components/reference/audio/multi-position/#multi-position-audio-properties.

### Submission Checklist:

* [X] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [X] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [X] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [X] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/components/reference/audio/switch.md [hunk 1]
```diff
@@ -11,13 +11,10 @@ The **Audio Switch** component provides basic [Audio Translation Layer (ATL)](/d
...
-The Audio Switch component has the following properties:
-
-**Default Switch**
-Enter the name of the audio switch to use by default. You can associate any audio switch with the entity.
-
-**Default State**
-Enter the name of the audio switch state to use by default. Use the [Audio Controls Editor](/docs/user-guide/interactivity/audio/audio-controls-editor) to assign the state to the switch. When this component is activated, the default switch is set to the default state.
+| Name | Description | Default |
+|------|-------------|---------|
+| Default Switch | Enter the name of the audio switch to use by default. You can associate any audio switch with the entity. | `<Empty>` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/audio/switch.md:16]:
```suggestion
| **Default Switch** | Enter the name of the audio switch to use by default. You can associate any audio switch with the entity. | `<Empty>` |
```

File: content/docs/user-guide/components/reference/audio/switch.md [hunk 2]
```diff
@@ -11,13 +11,10 @@ The **Audio Switch** component provides basic [Audio Translation Layer (ATL)](/d
...
+| Default State | Enter the name of the audio switch state to use by default. Use the [Audio Controls Editor](/docs/user-guide/interactivity/audio/audio-controls-editor) to assign the state to the switch. When this component is activated, the default switch is set to the default state. | `<Empty>` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/audio/switch.md:17]:
```suggestion
| **Default State** | Enter the name of the audio switch state to use by default. Use the [Audio Controls Editor](/docs/user-guide/interactivity/audio/audio-controls-editor) to assign the state to the switch. When this component is activated, the **Default Switch** is set to the **Default State**. | `<Empty>` |
```
Bold elements of the UI, including property names.  We can remind users that this property is affected by another property's value by using UI element's names with bolding again in the description.

**Review by FiniteStateGit — APPROVED** [review]

**Comment by LB-ArturZieba** [comment]
Sorry for the chaos with commits, I have a problem (similar to https://github.com/o3de/o3de.org/pull/1869), where including proposed changes returns a DCO fail, and then attempting to fix it following the GitHub instructions (`git rebase HEAD~X --signoff`, and `git push --force-with-lease origin <branch>`) causes all of the commits between my original Pull Request, and the rebase to be added. 

I have tried to add the `Signed-off-by: Artur Zięba 86952082+LB-ArturZieba@users.noreply.github.com` comment to the proposed updates as well but this does not help. Please let me know if this is an expected behavior, or if there is some way where I can accept the proposed changes while verifying the DCO, but without pinging all of the commits that happened in the meantime.

**Comment by FiniteStateGit** [comment]
@LB-ArturZieba The email may need angle brackets: <Signed-off-by: Artur Zięba 86952082+LB-ArturZieba@users.noreply.github.com>

**Comment by LB-ArturZieba** [comment]
Thanks for the answer, will try it out next time this occues. Should this Pull Request be redone (like https://github.com/o3de/o3de.org/pull/1869), or can this be properly merged?

**Comment by FiniteStateGit** [comment]
@LB-ArturZieba I would just resubmit it like #1869.

**Comment by LB-ArturZieba** [comment]
Closing this Pull Request as per above comment. A new Pull Request with relevant changes will be created instead.

**Pull request closed without merging**